src/locales/ja/parsers/JPCasualDateParser.ts

```diff
@@ -3,15 +3,38 @@ import dayjs from "dayjs";
 import { Meridiem } from "../../../index";
 import * as references from "../../../common/casualReferences";
 
-const PATTERN = /今日|当日|昨日|明日|今夜|今夕|今晩|今朝/i;
+const PATTERN = /今日|きょう|当日|とうじつ|昨日|きのう|明日|あした|今夜|こんや|今夕|こんゆう|今晩|こんばん|今朝|けさ/i;
+
+function normalizeTextToKanji(str: string) {
```

Comment:
Nit: `text` not `str`, because your function name is normalizeTextToKanji, not normalizeStrToKanji.

Reply:
Agree!